### DISCUSSION TITLE
sqlite Installation

### DISCUSSION THREAD

**Discussion by promexio** [givanz/Vvveb#32] [Q&A]
Hi all -
I'm trying to install it with the sqlite option, but it gives me an error:

`✖ Db error: "Unable to prepare statement: 1, table post_content_search already exists create virtual table post_content_search using fts5( content='post_content', content_rowid='rowid', name, content ); create virtual table post_content_search using fts5( content='post_content', content_rowid='rowid', name, content ); in post/post_content.sql" Error code: "0"`

I'm not sure what my mistake is and I can't find hints in the documentation.
Do I have to create sth specific or the .db file, if so, where do I have to place it?

Thanks & regards 
--promexio

**Comment by givanz** [comment]
Hi

Sorry, this issue is caused because the database file is not empty, it was zipped with data by mistake, I changed the zip and now the file is empty.

You can delete `storage/sqlite/vvveb.db` and create an empty file in it's place or just run something like

```bash 
truncate --size 0  /home/var/html/vvveb/storage/sqlite/vvveb.db
```

**Comment by promexio** [comment]
Thx for the quick reply!

Now, I get this:
`✖ Db error: "Unable to execute statement: no such module: fts5 create virtual table post_content_search using fts5( content='post_content', content_rowid='rowid', name, content ); create virtual table post_content_search using fts5( content='post_content', content_rowid='rowid', name, content ); in post/post_content.sql" Error code: "0"`

--
promexio

**Comment by givanz** [comment]
What version of sqlite do you have? fts5 module is the full text search module that is available starting with version 3.9.0

**Comment by promexio** [comment]
$ sqlite3 --version
3.7.17 

I guess, this is the problem. 
Any possibilities to move forward, ideas?

**Comment by givanz** [comment]
You can delete the full text search index by editing `install/sql/sqlite/schema/post/post_content.sql` and `install/sql/sqlite/schema/product/product_content.sql`  and delete everything starting with 

```sql 
create virtual table
``` 
until the end of file for both files and then rerun install.

This will break site search, if you need site search you can then edit `admin/sql/sqlite/post.sql` and `admin/sql/sqlite/product.sql` and comment or remove the full text search `MATCH` and uncomment the `LIKE` version.

```sql
AND pd.name LIKE CONCAT('%',:search,'%')
 -- AND (pcs.content MATCH :search) 
 ```
